backend/src/main/entity/Promotion.ts

```diff
@@ -144,4 +146,14 @@ export class Promotion {
     select: false,
   })
   tsVector: string;
+
+  /*
+   * Represents restaurant name inputted by user
+   * Many promotions can come from the same restaurant and thus have the same restaurantName
```

Comment:
Do you think it's okay to leave out this comment?

Reply:
Hmm yeah I can! I think it's nice to point out though?

Reply:
Ah wait yes my bad sorry looks good! 😅